### PR TITLE
added button component

### PR DESCRIPTION
**Pull request by Sharmaryan**

### CODE REVIEW

**Comment by Sharmaryan** [comment]
Screenshot (22)
Screenshot (23)
Screenshot (24)


added button, not responsive

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to merge:)

File: styles/buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  color: var(--secondary-color);
+}
+
+.btn-icons {
+  border: 1px solid var(--secondary-color);
+}
+
+.btn-desc {
+  margin: 0 6px;
+}
```
Comment (NeerajVodala) [inline styles/buttons/button.css]: using rem here is recommended( for consistency in code)

**Review by NeerajVodala — APPROVED** [review]
Looks good to me (change the unit before merging).
All the best.

**Review by Bhakti199 — APPROVED** [review]
looks cool, good to merge

File: styles/buttons/button.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/buttons/button.css]: ok i will do that

**Comment by Sharmaryan** [comment]
> Looks good to me (change the unit before merging). All the best.

done

**Pull request merged**